src/uct/ib/ud/base/ud_ep.h

```diff
@@ -222,7 +222,8 @@ struct uct_ud_ep {
          ucs_queue_head_t       window;       /* send window: [acked_psn+1, psn-1] */
          uct_ud_ep_pending_op_t pending;      /* pending ops */
          ucs_time_t             send_time;    /* tx time of last packet */
-         ucs_time_t             slow_tick;    /* timeout to trigger slow timer */
+         ucs_time_t             resend_time;  /* tx time of last resent packet */
```

Comment:
can we update send_time instead of keeping both send_time and resend_time?

Reply:
send_time is base for timeout, if resend will update it - timeout never happens

Reply:
right